metacat-common-server/src/main/java/com/netflix/metacat/common/server/events/MetacatEventBus.java

```diff
@@ -79,22 +72,6 @@ public void postAsync(final ApplicationEvent event) {
     public void postSync(final ApplicationEvent event) {
         log.debug("Received request to post an event {} synchronously", event);
         this.eventSyncCounter.increment();
```

Comment:
Improvement for later: We should tag these metrics with the type of the event. This will allow us to track what's going where and the scale of them. @zhljen has done a lot of this